doc/api/stream.md

```diff
@@ -1244,13 +1244,15 @@ changes:
     description: The `chunk` argument can now be a `Uint8Array` instance.
 -->
 
-* `chunk` {Buffer|Uint8Array|string|any} Chunk of data to unshift onto the
+* `chunk` {Buffer|Uint8Array|string|null|any} Chunk of data to unshift onto the
```

Comment:
This now says `null` is permitted, but lines 1248-1250 seem to say `null` is not permitted.

Reply:
Null is permitted, it will close the stream, but isn't a valid chunk in any mode, I don't have any idea of how to express it better

Reply:
Null is permitted, it will close the stream, but isn't a valid chunk in any mode, I don't have any idea of how to express it better